paddle/fluid/framework/tensor.cc

```diff
@@ -38,14 +42,20 @@ void* Tensor::mutable_data(const platform::Place& place,
                            proto::VarType::Type type, size_t requested_size) {
   type_ = type;
   PADDLE_ENFORCE_GE(numel(), 0,
-                    "When calling this method, the Tensor's numel must be "
-                    "equal or larger than zero. "
-                    "Please check Tensor::dims, or Tensor::Resize has been "
-                    "called first. The Tensor's shape is [",
-                    dims(), "] now");
+                    platform::errors::PreconditionNotMet(
+                        "The Tensor's numel must be "
+                        "equal or larger than zero. "
+                        "Please check Tensor::dims, or Tensor::Resize has been "
+                        "called first. The Tensor's shape is [",
+                        dims(), "] now"));
   size_t size = numel() * SizeOfType(type);
   if (requested_size) {
-    PADDLE_ENFORCE_GE(requested_size, size);
+    PADDLE_ENFORCE_GE(requested_size, size,
+                      platform::errors::InvalidArgument(
+                          "The requested memory size is less than the tensor."
```

Comment:
make sure `less than` and `lesser than`, which is correct grammar?

Reply:
done, thx